modules/ROOT/pages/amqp/amqp-connector.adoc

```diff
@@ -76,9 +76,9 @@ For example, we can define a prefetch size window in terms of size and in terms
 </amqp:config>
 ----
 
-In this case, every `consume` operation or `listener` associated to this config uses an acknowledge mode of type `IMMEDIATE`, while every time we do a `publish` or `publish-consume`, the outgoing Message requests a confirm from the broker.
+In the above example, every `consume` operation or `listener` associated to this config uses an acknowledge mode of the type `IMMEDIATE`, while every time we do a `publish` or `publish-consume`, the outgoing message requests a confirm from the broker.
```

Comment:
What is the object for the syntax `listener` (like "operation" is the object for `consume`)?

Also, ditto use of "config" vs. `config` or "configuration"?

Reply:
```suggestion
In the previous example, every `consume` operation or `listener` associated with this config uses an acknowledgment mode of the type `IMMEDIATE`, while for every `publish` or `publish-consume` operation, the outgoing message requests a confirmation from the broker.
```
Please adapt as necessary to ensure tech accuracy.